manifests/l/Libretro/RetroArch/Nightly/64-bit/1.10.3.0/Libretro.RetroArch.Nightly.64-bit.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Libretro.RetroArch.Nightly.64-bit
+PackageVersion: 1.10.3.0
```

Comment:
Wrong version. Should be 1.10.3

Reply:
Also, SHA256 sums are: 7fae3676a37a2b174692c9fc3d17b0c4db284966049e8630cf93a004aa588c60

Reply:
Not the right URL for the version 1.10.3, URL is https://buildbot.libretro.com/stable/1.10.3/windows/x86_64/RetroArch-Win64-setup.exe